### ISSUE TITLE
Tour of Heroes: Failed to load app.component.css

### ISSUE THREAD

**Issue by patelnav** [angular/angular#13433]
**I'm submitting a ...** 
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**

If you follow along the Tour of Heroes guide, you get a Failed to load app.component.css

**Expected behavior**

There shouldn't be bugs in the getting started guide!

**Minimal reproduction of the problem with instructions**

Make a module without a moduleId, you can't use styleUrls without a full relative path 

**What is the motivation / use case for changing the behavior?**

See expected behaviour.
Basically, Tour of Heroes tutorial should be updated to set the moduleId when styleUrls is set

**Please tell us about your environment:**
Warming

**Comment by DzmitryShylovich** [comment]
Pls report docs issues on https://github.com/angular/angular.io/issues
